Dockerfile.test

```diff
@@ -17,6 +17,7 @@ COPY shard.yml /app
 COPY shard.override.yml /app
 COPY shard.lock /app
 
+RUN shards update --ignore-crystal-version
```

Comment:
```suggestion
```

This will result in an update to the `shard.lock`, and the resulting shards will be different.
The `shard.lock` is there to ensure that the set of dependencies installed are reproducible